components/homme/src/share/gllfvremap_mod.F90

```diff
@@ -138,6 +140,11 @@ module gllfvremap_mod
      module procedure gfr_dyn_to_fv_phys_dom_mt
   end interface gfr_dyn_to_fv_phys
 
+  interface gfr_potvort_dyn_to_fv_phys
```

Comment:
We should follow the same naming convention as the rest of these routines: "`gfr_dyn_to_fv_phys_<quantity>_<hybrid,dim_mt>`"

Reply:
done